lib/readGltf.js

```diff
@@ -10,14 +10,13 @@ var DeveloperError = Cesium.DeveloperError;
 
 module.exports = readGltf;
 
-function readGltf(gltfPath, callback) {
+function readGltf(gltfPath, options, callback, imageProcess) {
     if (!defined(gltfPath)) {
         throw new DeveloperError('Input path is undefined.');
     }
     
     var fileExtension = path.extname(gltfPath);
-    var filePath = path.dirname(gltfPath);
```

Comment:
I know this change is now everywhere, but I think `options.basePath` should just be an optional argument for `readGltf`. Almost all of the time the `basePath` can be computed like this.